TicTacToe/SettingController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  SettingController.swift
+//  TicTacToe
+//
+//  Created by Tahia Ata on 5/19/16.
+//  Copyright © 2016 Tahia Ata. All rights reserved.
+//
+
+import UIKit
+
+class SettingController: UITableViewController {
+
+    let playerCellIdentifier = "PlayerCell"
+    let defaultImageCellIdentifier = "DefaultImageCell"
+    let backGroundImageCellIdentifier = "BackGroundImageCell"
+    let totalSettingItem = 4
```

Comment:
Howerver, in case data type is implicit, specification is necessary.

// ◯ Int / UInt / NSInteger / NSUInteger these data types should be distinguished so specification is necessary
let i: Int = 0

// ◯ Float / CGFloat / Double these data types should be distinguished so specification is necessary
let p: Float = 3.14